src/recipes/scivolo.json

```
@@ -0,0 +1,25 @@
+{
+    "name": "Scivolo",
+    "description": "Sweet Italian cocktail",
+    "github": "alepit",
+    "ingredients": [
+        {
+            "quantity": "1 1/3",
+            "measure": "oz",
+            "ingredient": "Strawberry Vodka"
+        },
+        {
+            "quantity": "2 1/3",
+            "measure": "oz",
+            "ingredient": "Lemon Schweppes"
+        }
+
+    ],
+    "directions": [
+        "Fill a glass with ice and pour in vodka and schweppes",
+        "Mix",
+        "Garnish with a lime or lemon slice."
+    ],
+    "image": "scivolo.png",
```

```suggestion
    "image": "scivolo.jpg",
```

Also, please remove the old .png image.